src/content/docs/cms/sanity/sanity-case-study.md

```diff
@@ -27,6 +27,59 @@ Over the years, the Sanity team has continued to improve and expand the platform
+2.	apicdn.sanity.io: the CDN-distributed, cached API, which is an opt-in feature that will give you very fast responses to requests that have been cached. We encourage most users to use this API for their front-ends if you expose your API to end users. For static builds you are better off using the live uncached API, to always get the latest version.
+To use the API CDN, simply use apicdn.sanity.io instead of api.sanity.io. Most clients provide a use CDN option that makes this switch seamless.
+
+
+Today we can use 20 frameworks, being React and Next.js the more popular.    
+You do not need to use php, Sanity uses Groq which is Sanity’s open-source query language.  It is possible to use graphQL if preferred. 
+When adding images to projects it’s possible to use a plug in, a powerful pipeline that  can take care of resizing, storage, deletion. 
+
+The amount of plugins and tools is also increasing, 210 available per today. Sanity names some “must-haves” plugins:  
+-	Scheduled publishing, to organize upcoming releases. 
+-	Media browser, a way of managing and organizing assets. 
+-	Vision (GROQ playground, for testing GROQ queries). 
+-	Image URL, to customize and optimize images
+-	The official Sanity toolkit for Next.js, to integrate content from Sanity
+-	Content Graph View, to see changes in real-time
+
+### Some features in Sanity: 
+
+Structured Content 101helps keeping a clean and organized content. It is content broken down into small pieces. This provides information that the machine can understand and use in all kinds of ways. 
+
+Real time update-This feature is helpful when collaborators make changes and those are updated in real time. 
+
+Presenting and previewing content.This gives the possibility of previewing content changes before publishing.  This is a useful tool when getting feedback and approval, minimizing errors.  
+Composable content allows shaping data from independent but interconnected documents. Drasj documents can co-exist with published versions of the same document. 
+Perspectives let you run queries “draft” and run a preview of the changes before  drafts publishes. This can help to avoid conflicts, and accidental publications. 
+
+
+When creating documents or editing published documents, a draft is create. Drafts keeps changes separated from published documents until those changes are good to go. 
+When drafts are published they become available on the public APIs and Strong references are important to avoid confusion with other documentation. 
+Sometimes we may want to get changes published right away, without using drafts. This is possible including liveEdit: true in the shema definition: 
+
+```
+export default {
+  name: 'author',
+  title: 'Author',
+  type: 'document',
+  liveEdit: true,
+  …
+```
+
+You can descover much more on reference docs 
```

Comment:
Great job! Looks good overall. I would say that maybe we can try to reword it a bit so that it flows a bit better :)
Once that is done I say we could merge it!